docs/CONFIGURATION.md

```diff
@@ -141,6 +141,10 @@ The agent table configures Telegraf and the defaults used across all plugins.
 - **quiet**:
   Log only error level messages.
 
+- **log_target**:
+  Log target - `file`, `stderr`, `stdout` or `eventlog` (Windows only). 
```

Comment:
If we are adding new option which controls the log target, wouldn't be nice to let it be the main controller of log target and have the other option (logfile) as a additional log config values param?

Reply:
I guess this would be good.  In a sense we would be deprecating `logfile="" = stderr` which was never great.

Reply:
How to properly inform about deprecation? For sure, I will add deprecation to comment of the logfile param to configuration code and the doc. I saw that some other plugins also print warnings.
So, if config will not contain `logTarget` and it will find `logfile=""` it should print warning about deprecation and info about new option?

Reply:
Let's not formally deprecate it yet, we can do this later.